Add function to check RKNN quantization in JNI

boomermath opened the pull request:
Adds a function to the RKNN JNI API to check if a model is quantized or not

boomermath committed:
add is quantized check

boomermath committed:
add non quant model to check

boomermath committed:
actually load the lib lol

boomermath committed:
add non quant model to check

mcm001 commented on src/main/native/cpp/rknn_jni.cpp:
```diff
@@ -25,32 +25,31 @@
-    std::printf("Couldn't find class!");
-    return JNI_ERR;
-  }
+    if (!detectionResultClass) {
```
Revert unrelated formatting diff

mcm001 commented on src/test/java/org/photonvision/rknn/RknnTest.java:
```diff
@@ -53,17 +45,17 @@ public void testBasicBlobs() {
         System.out.println(Core.getBuildInformation());
         System.out.println(Core.OpenCLApiCallError);
 
-         
```
Revert (goes for the rest of the diff)

mcm001 commented on src/test/java/org/photonvision/rknn/RknnTest.java:
```diff
@@ -18,31 +18,23 @@
+
 public class RknnTest {
-    
+    private static final String CWD = "/home/boomermath/rknn_jni";
```
Don't hard code

mcm001 commented on src/main/native/include/yolo_common.hpp:
```diff
@@ -18,6 +18,8 @@ enum ModelVersion {
 
 class YoloModel {
 public:
+    bool is_quant;
```
Why did this move to the top?

boomermath replied:
Make it accessible for JNI. Would you prefer I use a getter instead?

mcm001 replied:
What's the convention for the order of functions and members for C++?

boomermath committed:
fix issues hopefully?

boomermath committed:
fix member ordering

boomermath committed:
resolve merge conflicts

boomermath committed:
im hoping this rebase works

boomermath committed:
Add linting workflow (#27)

# Conflicts:
#	src/main/java/org/photonvision/rknn/RknnJNI.java
#	src/main/native/include/yolo_common.hpp
#	src/test/java/org/photonvision/rknn/RknnTest.java
